haystack/evaluation/eval.py

```diff
@@ -175,8 +182,106 @@ def _calculate_em(
 
         return MetricsResult({"exact_match": exact_match_score})
 
-    def _calculate_sas(self):
-        return MetricsResult({"exact_match": None})
+    def _calculate_sas(
+        self,
+        output_key: str,
+        regexes_to_ignore=None,
+        ignore_case=False,
+        ignore_punctuation=False,
+        ignore_numbers=False,
```

Comment:
Can we add some times to these arguments? These should be correct I think?

```suggestion
        regexes_to_ignore: Optional[List[str]] = None,
        ignore_case: bool = False,
        ignore_punctuation: bool = False,
        ignore_numbers: bool = False,
```

Reply:
Yes, these are correct. I will update the other metrics too.

Reply:
Feel free to do it in another PR, this is good to merge in my opinion as of now. :)

Reply:
I already pushed the changes here, should I revert them?

Reply:
No worries, all good if they're here. 👍
Just wanted to avoid any linting issue preventing merging.

Reply:
Thanks!